src/iosX64Test/kotlin/de/voize/pytorch_lite_multiplatform/TorchModuleIOSTest.kt

```diff
@@ -17,89 +18,128 @@ class TorchModuleIOSTest {
 
     @Test
     fun itCanRunMethod() {
-        val module = TorchModule(localModulePath)
-        val output = module.runMethod("inference", listOf(FloatTensor(
-            FloatArray(10) { 0.0F },
-            longArrayOf(1, 10),
-        )))
-        assertEquals(10, output.data.size)
-        assertContentEquals(longArrayOf(1, 10), output.shape)
-    }
-
-    @Test
-    fun itCanRunMethodWithDictInput() {
-        val module = TorchModule(localModulePath)
-        val output = module.runMethod(
-            "inference_dict",
-            mapOf(
-                "x" to FloatTensor(
-                    FloatArray(10) { 0.0F },
-                    longArrayOf(1, 10),
-                )
+        memScoped {
```

Comment:
@Legion2 @iceychris this is not a common-compatible API.
This should be `PLMScope` which behaves like `memScoped` but with a noop on Android

Reply:
https://kotlinlang.slack.com/archives/C3SGXARS6/p1659114778164219